.gitignore

```diff
@@ -2,3 +2,6 @@
 .obj
 cmake-build-debug
 .idea/workspace.xml
+build
+ICLFilter/src/ICLFilter/OpenCL/BilateralFilterOpKernel.h

```

Comment:
These files shouldn't be generated in source tree. This should be fixed later...